airflow/contrib/hooks/gcp_vision_hook.py

```diff
@@ -515,6 +519,7 @@ def document_text_detection(
         return response
 
     @GoogleCloudBaseHook.catch_http_exception
+    @GoogleCloudBaseHook.quota_retry()
```

Comment:
Is () necessary in the decorator? If not, should be consistent and drop it?

Reply:
It's neccesary, because quota_retry allows you to provide a custom retry rules.